student.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
+    super(age, name)
+    @classroom = classroom
+    @age = age
+    @name = name
+  end
```

Comment:
[OPTIONAL] It would be best to have `age` and `name` as parameters in the `initialize` method in order to be able to add new student information. Otherwise the values for `name` and `age` will always be undefined.

Reply:
😄 Done